lib/postcode_validation/use_case/format_validators/be_postcode_validator.rb

```diff
@@ -0,0 +1,6 @@
+class BEPostcodeValidator
```

Comment:
Ok, so the country specific file only contains the regex and then a separate RegexValidator validates the postcode and the FormatValidator selects the country specific file based on the country param?